src/python/pants/engine/internals/native.py

```diff
@@ -55,14 +56,32 @@ def create_exception(self, msg):
         """Given a utf8 message string, create an Exception object."""
         return Exception(msg)
 
-    def generator_send(
-        self, func, arg
-    ) -> Union[PyGeneratorResponseGet, PyGeneratorResponseGetMulti, PyGeneratorResponseBreak]:
+    def val_to_str(self, val):
```

Comment:
I just removed this method in an earlier commit today, so I think this is extraneous

Reply:
Yes! Just killed it!

Reply:
Yes! Just killed it!

Reply:
(in ab10341)